fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -2724,31 +2724,6 @@ public void testGraphShouldNotBeTooLumpy() throws IOException {
 
     }
 
-    @Test
-    public void testEmbeddedChildResources() throws IOException {
```

Comment:
I believe this test (and the others in this PR) are good... they just need to be changed for the new `Prefer` header, no?

Reply:
I think we've got a similar test testEmbeddedContainedResources() in https://github.com/fcrepo4/fcrepo4/pull/1290/files/ef6e3ccf120bdc6c1ff36679db7cdccd689633b9#diff-39785d72ffe23548cc619403f35d9028L2753.